<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="filter_free_text_la">
       <property name="text">
        <string>Free text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="free_text_le"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="advanced_filters_gb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Advanced Filters</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="filter_id_la">
          <property name="text">
           <string>Identifier(s)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="id_le"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="filter_property_name_la">
          <property name="text">
           <string>Property name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="property_name_le"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="filter_property_value_la">
          <property name="text">
           <string>Property value</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="property_value_le"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QgsExtentGroupBox" name="spatial_extent_gb">
          <property name="title">
           <string>Spatial Extent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="temporal_extent_gb">
          <property name="title">
           <string>Temporal Extent</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateEdit" name="temporal_extent_start_de"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>End</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="temporal_extent_end_de"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
